testdata/goldens/dotnet/Google.Cloud.Asset.V1.AnalyzeIamPolicyResponse.Types.IamPolicyAnalysis.Types.Stats.Types.NodeType.html

```diff
@@ -14,9 +14,11 @@ <h1 class="page-title">Enum AnalyzeIamPolicyResponse.Types.IamPolicyAnalysis.Typ
   <div class="markdown level0 summary"><p>Type of the node.</p>
 </div>
   <div class="markdown level0 conceptual"></div>
-  <h6><strong>Namespace</strong>: <a class="xref" href="Google.Cloud.Asset.V1.html">Google.Cloud.Asset.V1</a></h6>
-  <h6><strong>Assembly</strong>: Google.Cloud.Asset.V1.dll</h6>
-  <h5 id="Google_Cloud_Asset_V1_AnalyzeIamPolicyResponse_Types_IamPolicyAnalysis_Types_Stats_Types_NodeType_syntax">Syntax</h5>
+  <h2>Namespace</h2>
+  <a class="xref" href="Google.Cloud.Asset.V1.html">Google.Cloud.Asset.V1</a>
+  <h2>Assembly</h2>
+  Google.Cloud.Asset.V1.dll

```

Comment:
@jskeet should we keep Assembly?

Reply:
Yes, I'd expect to see that.

Reply:
@dandhlee I think `Google.Cloud.Asset.V1.dll` needs to be in a `<p>`?

Reply:
done.